content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -68,8 +68,9 @@ visible in the attachment section, or embedded in the PDF.
 .. note::
    - For E-FFF, the xml file only appears after having generated the PDF (:guilabel:`Print` or
      :guilabel:`Send & Print` button), since the PDF needs to be embedded inside the xml.
-   - By default, the :guilabel:`Factur-X` option is enabled. It means that an XML file is
-     automatically included in the PDF document that is sent.
+   - Every PDF generated from Odoo contains a :guilabel:`Factur-X` XML file (for interoperability purpose).
+     For German and French companies, the option :guilabel:`Factur-X (PDF/A-3)` in addition enables
+     validation checks on the invoice and generates a PDF/A-3 compliant file, required by plaftorms like Chorus Pro.
```

Comment:
- grammar and typos
- break the line at 100th character (see [RST guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))
- the first Factur-X doesn't require a [guilabel markup](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#gui-element) since it's not an element on-screen anymore
- split into two different points

```suggestion
   - Every PDF generated from Odoo contains a **Factur-X XML file** for interoperability purposes.
   - For German and French companies, the format :guilabel:`Factur-X (PDF/A-3)` is available in
     addition to Odoo's default Factur-X. This format enables validation checks on the invoice and
     generates a **PDF/A-3** compliant file, as required by platforms such as Chorus Pro.
```